staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -3148,3 +3148,44 @@ func TestListIndexer(t *testing.T) {
 		})
 	}
 }
+
+func TestRetryAfterForUnreadyCache(t *testing.T) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.ResilientWatchCacheInitialization) {
+		t.Skipf("the test requires %v to be enabled", features.ResilientWatchCacheInitialization)
```

Comment:
alternatively if the feature is disabled we could check if the expected retryAfter is always `1`.